Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.cxx

```diff
@@ -1414,7 +1435,9 @@ bool vtkMRMLMarkupsNode::ResetNthControlPointID(int n)
     {
     return false;
     }
+
   this->SetNthControlPointID(n, this->GenerateUniqueControlPointID());
+  this->LastUsedControlPointNumber++;
```

Comment:
This manual incrementing of LastUsedControlPointNumber is quite messy. I think it should be done in GenerateUniqueControlPointID() method.

Reply:
I also thought about this, but `LastUsedControlPointNumber` is not systematically incremented each time GenerateUniqueControlPointID is called ...

Reply:
This case is the only one where it (currently) isn't incremented?
https://github.com/Slicer/Slicer/blob/d3bd8dd0377fd437ad87f0cd6b288c57222571d3/Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.cxx#L482

Is this correct or a bug?